src/main/java/io/vinyldns/java/model/zone/GetZoneRequest.java

```diff
@@ -0,0 +1,35 @@
+package io.vinyldns.java.model.zone;
+
+public class GetZoneRequest {
+    private String zoneId;
```

Comment:
mark this as final `private final String zoneId`

Reply:
can't without initializing no?